pkg/util/deployer/deployer.go

```diff
@@ -0,0 +1,125 @@
+package deployer
```

Comment:
Could you please provide package docstring explaining what the deployer does and how to use it? Thank you.